WordPress/src/main/java/org/wordpress/android/util/DialogUtils.java

```diff
@@ -12,6 +12,9 @@
 import org.wordpress.android.widgets.WPTextView;
 
 public class DialogUtils {
+
+    private static OpenSansEditText editText;
```

Comment:
I think it's much more cleaner having a callback that saves the last inserted value in `MyProfileActivity`. 
That way we can save/restore the last inserted value like all the other fields in `onCreate`/`onSaveInstanceState`.